### PR TITLE
Add Qtum support

### PR DESCRIPTION
**Pull request by icodeface**

### CODE REVIEW

File: defs/bitcoin/qtum.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "coin_name": "Qtum",
+    "coin_shortcut": "QTUM",
+    "coin_label": "Qtum",
+    "website": "https://qtum.org",
+    "github": "https://github.com/qtumproject/qtum/",
```
Comment (matejcik) [inline defs/bitcoin/qtum.json]: please strip the trailing `/`, here and in testnet

Comment (icodeface) [reply defs/bitcoin/qtum.json]: fixed

File: defs/bitcoin/qtum.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "minfee_kb": 400000,
+    "signed_message_header": "Qtum Signed Message:\n",
+    "hash_genesis_block": "000075aef83cf2853580f8ae8ce6f8c3096cfa21d98334d6e3f95e5582ed986c",
+    "xprv_magic": 76066276,
+    "xpub_magic": 76067358,
+    "xpub_magic_segwit_p2sh": 77429938,
+    "xpub_magic_segwit_native": 78792518,
+    "bech32_prefix": "qc",
+    "cashaddr_prefix": null,
+    "slip44": 88,
```
Comment (matejcik) [inline defs/bitcoin/qtum.json]: per [SLIP-44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) this value should be 2301

**Comment by matejcik** [comment]
not sure why you removed the duplicate-overrides entry right now?

File: defs/bitcoin/qtum.json (hunk 2 above)
Comment (icodeface) [reply defs/bitcoin/qtum.json]: Hi @matejcik, the 2301 is an incorrect number, see the discussion here https://github.com/satoshilabs/slips/pull/196

**Comment by icodeface** [comment]
> not sure why you removed the duplicate-overrides entry right now?

I just saw 

> External contributors should not make changes to `duplicity_overrides.json`, unless
> asked to.

in the https://github.com/trezor/trezor-common/blob/master/defs/README.md

File: defs/bitcoin/qtum.json (hunk 2 above)
Comment (matejcik) [reply defs/bitcoin/qtum.json]: could be, but per the same discussion and https://github.com/satoshilabs/slips/pull/120 number 88 is already taken and we don't allow duplicates in our codebases.

**Comment by matejcik** [comment]
well spotted, but in this case I'd be making the exact same change, so please go ahead and put it back there :)

File: defs/bitcoin/qtum.json (hunk 2 above)
Comment (icodeface) [reply defs/bitcoin/qtum.json]: changed to 2301

**Comment by matejcik** [comment]
merged, thanks

**Pull request merged**